.gitignore

```diff
@@ -1,5 +1,13 @@
 _build/
 _site/
+Properties/
+node_modules/
+wwwroot/
+
+#override .sln when required. Not needed in most cases.
+*.sln
```

Comment:
I prefer not to include .sln here.  There are times when you need it, and overriding a .gitignore is going to be a nuisance, and it doesn't hurt anything if it's there when you don't need it.